src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,105 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int DEFAULT_AGE = 18;
+    private static final int DEFAULT_PASSWORD_LENGTH = 6;
+    private static final int STORAGE_SIZE_FOR_TEST = 2;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user;
+
+    @BeforeEach
+    public void defaultUser() {
+        user = new User();
+        user.setId(123L);
+        user.setLogin("vasyl");
+        user.setPassword("vasyl55555");
+        user.setAge(24);
+    }
+
+    @Test
+    void check18_years_old_NotOk() {
```

Comment:
```suggestion
    void register_lowAge_notOk() {
```

Reply:
Go through all class and rename methods according to task
![image](https://user-images.githubusercontent.com/20326858/201944173-eef1548a-c97e-4692-a856-a1c71ef4a904.png)